eth/v1alpha1/node.proto

```diff
@@ -62,47 +47,54 @@ service Node {
             get: "/eth/v1alpha1/node/services"
         };
     }
-
-    // Retrieve the list of peers currently connected to this node.
-    rpc ListPeers(google.protobuf.Empty) returns (Peers) {
-        option (google.api.http) = {
-            get: "/eth/v1alpha1/node/peers"
-        };
-    }
-}
-
-// Information about the current network sync status of the node.
-message SyncStatus {
-    // Whether or not the node is currently syncing.
-    bool syncing = 1;
 }
 
-// Information about the genesis of Ethereum 2.0.
-message Genesis {
-    // UTC time specified in the chain start event in the deposit contract.
-    google.protobuf.Timestamp genesis_time = 1;
-
-    // Address of the deposit contract in the Ethereum 1 chain.
-    bytes deposit_contract_address = 2;
+message NodeInfo {
+    // A unique identifier for the node.
+    string node_id = 1;
```

Comment:
might be useful to detail under which circumstances this could change, if it can.

Reply:
Good point.  The only thing we have in the codebase that identifies the node is its network key, from which its peer ID is calculated.  This can change if the underlying network key is changed.  I'll add the derivation information.